packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-phone/java/summaryofsteps.md

```diff
@@ -0,0 +1,9 @@
+## Summary of steps
+
+The following graphic shows the steps for the sign-in flow with password and phone factors:
+
+<div class="common-image-format">
+
+![Displays the sequence of steps for the Java SDK sign in with password and phone flow](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-phone-seq-1-java.png)
```

Comment:
Same comment as before about consistency of the diagram method calls.

Reply:
One more thing, I had a look, and I think most of the comments I made here on the password/phone factors article are also true for the password/email factors article.